tests/acceptance/user-list-test.js

```diff
@@ -23,20 +23,25 @@ var page = PO.build({
     itemScope: 'tbody tr',
     item: {
       userName: PO.text('td', { index: 1 }),
-      role: PO.text('td', { index: 2})
+      role: PO.text('td', { index: 2}),
+      selectGender: PO.selectable('select'),
+      gender: PO.text('select option:selected')
```

Comment:
I think the right way of getting the selected options is using `PO.value`

We're using `jQuery.val()` under the hood so:
https://learn.jquery.com/using-jquery-core/faq/how-do-i-get-the-text-value-of-a-selected-option/